runtime/views.py

```diff
@@ -20,23 +20,23 @@
 def tracesList(self):
     traces = XTrace.objects.all()
     serializer = TraceSerializer(traces, many=True)
-    return Response(serializer.data, status=201)
+    return Response(serializer.data, status=200)
```

Comment:
why this change?

Reply:
only because with a get call I preferred to return OK code instead of create code, but nothing in particular